tests/reference-01/test.yaml

```diff
@@ -0,0 +1,19 @@
+requires:
+  min-version: 8
+
+args:
```

Comment:
This pcap already exists in `dcerpc/zerologon`

Reply:
Thanks. Updated.